goalie/mesh_seq.py

```diff
@@ -527,6 +527,9 @@ def solve_forward(self, solver_kwargs: dict = {}) -> AttrDict:
         P = self.time_partition
         solver = self.solver
 
+        # Reinitialise the solutions dictionary
```

Comment:
Again

Reply:
Fixed in e8d888f3ecc9c0590811374e67cc421b5d175cfb